x-pack/legacy/plugins/apm/public/components/shared/charts/Timeline/TimelineAxis.js

```diff
@@ -15,26 +15,26 @@ import { px } from '../../../../style/variables';
 import { getTimeFormatter } from '../../../../utils/formatters';
 import theme from '@elastic/eui/dist/eui_theme_light.json';
 
-// Remove any tick that is too close to traceRootDuration
-const getXAxisTickValues = (tickValues, traceRootDuration) => {
-  if (traceRootDuration == null) {
+// Remove any tick that is too close to topTraceDuration
+const getXAxisTickValues = (tickValues, topTraceDuration) => {
+  if (topTraceDuration == null) {
```

Comment:
What is `topTraceDuration`? I think we should be careful introducing new terminology unless necessary. 
Isn't `topTraceDuration` just `traceDuration` (and perhaps `duration` for some components)?

Reply:
I've heard it mentioned as `entryTransaction`. And we do use that term (`entryTransactionId`). So this would be `entryTransactionDuration`. 

Personally I don't think `entryTransactionDuration` sounds better than `topTraceDuration` but we should stick to one :)

Reply:
Another options is `waterfallDuration` since it comes from `waterfall.duration`.

Reply:
It's different than the trace root, so I had to come up with something new, but I like both `waterfallDuration` the most, I think that's the easiest to understand for the reader of this code.

Reply:
Cool, `waterfallDuration` it is then.
I saw you asked the other day about APM terminology. If you have suggestions for improving it I'm all ears. The terms have evolved a bit since we started so some of it might be outdated. Just like code I think it's important to refactor terminology once in a while so it stays up to date.